forward43/scraper_kickstarter.py

```diff
@@ -53,16 +65,22 @@ def scrape(self):
             for page in range(1, self.num_pages + 1):
                 self.logger.info(f'Processing category: {category} and page: {page}')
 
+                # Now, instead of trying a single url, it tries to loop over the list created above
                 try:
-                    url       = self.get_url(category, page)
-                    response  = self.get_response(url)
-                    projects  = self.process_response(response)
-
+                    url_term_list       = self.get_url(category, page)
+                    
+                    print(url_term_list)
```

Comment:
use the logger if you want to log what it's doing, otherwise remove the print statement